contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/common.py

```diff
@@ -91,15 +91,15 @@ def _remove_coding_header(cls, blob):
     """
     # Remove the # coding=utf-8 to avoid AST erroneous parse errors
     #   https://bugs.python.org/issue22221
-    lines = blob.split('\n')
+    lines = blob.decode('utf-8').split('\n')
```

Comment:
I kept `blob` always being binary string—even though I don't think it's necessary to read these source files as binary—to stay closer to original semantics.

`lines` are always unicode now. Earlier it was ambiguous what they were.